README.md

```diff
@@ -50,6 +50,17 @@ The current version of media includes the following functionality:
 * Ability to embed tweets, Instagrams, and YouTube/Vimeo videos directly into
   CKEditor by pasting the video URL
 
+#### Extending Lightning Media (Contributed Modules)
+A number of contributed modules have been written by Drupal community to integrate Drupal Media Entity with Lightning. This allows a seamless integration between Lightning functionality and 3rd party media services via Media Entity. Below is a list of 3rd party services and contributed modules that are currently available for Lightning Media:
```

Comment:
Let's streamline this a little bit. How about:

"Drupal community members have contributed several modules which integrate Lightning Media with additional third-party media services. These modules are not packaged with Lightning or maintained by Acquia, but they are stable and you can use them in your Lightning site:"